frontend/src/app/[locale]/events/EventsUpcoming.tsx

```diff
@@ -8,32 +8,28 @@ export default function EventsUpcoming() {
   const t = useTranslations("Events.upcoming");
 
   return (
-    <GridContainer data-testid="events-upcoming-content" className="padding-4">
-      <Grid row gap="md">
-        <Grid
-          tablet={{
-            col: 3,
-          }}
-        >
-          <h1 className="margin-left-6">{t("title")}</h1>
+    <GridContainer
+      data-testid="events-upcoming-content"
+      className="padding-y-4 tablet-lg:padding-y-6"
+    >
+      <Grid row gap>
+        <Grid tablet={{ col: 4 }}>
+          <h2>{t("title")}</h2>
```

Comment:
- Use responsive grid gap
- Use column widths consistent with other pages/sections
- Heading is for section not page (reduce nesting level)